python/oauth-demo/requirements.txt

```diff
@@ -0,0 +1,778 @@
+acme==2.6.0
+aiofiles==24.1.0
+aiohttp==3.8.4
+aioitertools==0.12.0
+aiosignal==1.3.1
+altair==5.4.1
+amazon_kclpy==2.1.5
+ansicolors==1.1.8
+anthropic==0.32.0
+antlr4-python3-runtime==4.13.2
+anyio==4.6.0
+appdirs==1.4.4
+appnope==0.1.2
+argcomplete==3.5.1
+asana==5.0.10
+astroid==2.9.3
+asttokens==2.4.1
+async-timeout==4.0.3
+asyncssh==2.17.0
+attrs==23.2.0
+awacs==2.4.1
+aws-kinesis-agg==1.2.3
+aws-lambda-powertools==2.43.1
+aws-sam-translator==1.91.0
+aws-xray-sdk==2.14.0
+awscli==1.29.41
+babel==2.16.0
+backcall==0.2.0
+backports.functools-lru-cache==1.6.6
+basictracer==3.2.0
+bazel_runfiles==0.36.0
+beartype==0.17.0
+beautifulsoup4==4.12.3
+black==23.7.0
+blinker==1.8.2
+Bootstrap-Flask==2.4.1
+boto==2.49.0
+boto3==1.35.36
+boto3-stubs==1.35.36
+botocore==1.35.36
+botocore-stubs==1.35.36
+Brotli==1.1.0
+build==1.0.0
+cached-property==1.5.2
+cachetools==5.5.0
+Cerberus==1.3.5
+certbot==1.1.0
+certbot-dns-route53==1.1.0
+certifi==2024.8.30
+cffi==1.17.1
+cfn-flip==1.3.0
+cfn-lint==1.16.0
+chardet==5.2.0
+charset-normalizer==3.3.2
+chkcrontab==1.7
+click==8.1.7
+cloudpickle==2.2.1
+codeguru_profiler_agent==1.2.5
+colorama==0.4.6
+commonmark==0.9.1
+ConfigArgParse==1.7
+configobj==5.0.8
+configparser==4.0.2
+contextdecorator==0.10.0
+contextlib2==0.6.0.post1
+contourpy==1.3.0
+crontab==0.23.0
+crowdstrike-falconpy==1.4.5
+cryptography==42.0.8
+cycler==0.12.1
+databricks-connect==13.3.3
+databricks-sdk==0.34.0
+dataclasses-json==0.5.8
+datadog==0.49.1
+datadog-api-client==2.29.0
+decorator==5.1.1
+defusedxml==0.7.1
+Deprecated==1.2.14
+deprecation==2.1.0
+dill==0.3.9
+distlib==0.3.8
+distro==1.9.0
+dnspython==2.6.1
+docker==6.1.3
+docutils==0.16
+dropbox==9.5.0
+ec2-metadata==2.13.0
+ecdsa==0.18.0
+elasticsearch==7.17.12
+enum-compat==0.0.3
+et-xmlfile==1.1.0
+exceptiongroup==1.2.2
+executing==2.1.0
+fakeredis==1.10.2
+fieldanalysis @ file:///Users/andrewwong/Desktop/custom-field-analysis/dist/fieldanalysis-0.1.0.tar.gz
```

Comment:
Hmmm, this doesn't look like it belongs here. Also, there are a lot of packages in this `requirements.txt` file I am assuming not all of them are needed since you specified a short list of dependencies in the `README.md`